angucomplete-alt.js

```diff
@@ -65,6 +66,9 @@
     );
 
     function link(scope, elem, attrs, ctrl) {
+	    
+	    console.log( scope.readOnlyInput );
```

Comment:
Oh man - I’ll clear that up.  Thanks!


> On May 2, 2016, at 11:49 AM, Loren <notifications@github.com> wrote:
> 
> In angucomplete-alt.js <https://github.com/ghiden/angucomplete-alt/pull/386#discussion_r61757510>:
> 
> > @@ -65,6 +66,9 @@
> >      );
> >  
> >      function link(scope, elem, attrs, ctrl) {
> > +	    
> > +	    console.log( scope.readOnlyInput );
> It looks like you may have accidentally left some test code in the file.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/ghiden/angucomplete-alt/pull/386/files/eea7713c553494804586bbd217cf5275065e4d1c#r61757510>